#subscriber
sub_assigned_virtual_position_topic: /uav2/teaming_planner/assigned_virtual_position

#publisher
pub_to_sense_avoid_topic: /uav2/sense_avoid/setpoint_position

source_frame: /odom
target_frame: /uav2

setpoint_z: 2.0

rate: 15
